pkg/k8s/deployments/translate.go

```diff
@@ -238,20 +258,3 @@ func translateOktetoVolumes(d *appsv1.Deployment, dev *model.Dev) {
 		d.Spec.Template.Spec.Volumes = append(d.Spec.Template.Spec.Volumes, v)
 	}
 }
-
-func getPullPolicy() apiv1.PullPolicy {
```

Comment:
Are we removing the support for this? I thought this was breaking quid's use ase

Reply:
I talked to alberto the other day. he said we should keep the policy defined in the deployment manifest. It is what we are doing for other fields, I think we should follow that approach here too.